media/css/firefox/whatsnew/whatsnew-67.0.5.scss

```
@@ -0,0 +1,210 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/hero';
+@import '../../protocol/components/state-fxa';
+
+body {
+    background: url('/media/img/firefox/whatsnew_67/noodles.png') no-repeat;
```

```suggestion
    background: $color-white url('/media/img/firefox/whatsnew_67/noodles.png') no-repeat;
```
TL;DR: add background colour declaration please.

Okay, reeeeeeeealy obscure edge case to consider here. When declaring the background shorthand if you don't set a background-colour it will set the colour to transparent. Back in the day browsers used to have a default background colour of grey. So this could have resulted in a grey background. I bring this up now because of the recent appearance of dark themes. I don't actually know if this will cause problems but I think it might be wise to be explicit here.

---

👏 super interesting! will do.